build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -1,5 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildBinPath)\Microsoft.Common.targets" />
+  <PropertyGroup>
+    <BuildDepends>
```

Comment:
> is the expectation now that a dev should use _Build to do an actual build "in all cases"?

That depends on our definition of "a dev." The *only* time they would need to use `_Build` is when:

1. They're working on **mono**, *not* Xamarin.Android, *and*
2. They haven't locally "bumped" the mono commit.

Developers working on Xamarin.Android need not *ever* build mono (so long as there's a bundle for it).

The #runtime team *may* need the `_Build` target, *if* they haven't done a `git pull` from `external/mono` first, as the timestamp checks won't otherwise catch their changes. If they *do* `git pull` first, then `Build` will call `_Build` implicitly.